core/src/common/lwm2m_plaintext.c

```diff
@@ -291,10 +291,10 @@ static int PTDeserialiseResource(SerdesContext * serdesContext, Lwm2mTreeNode **
 // Map PlainText serdes function delegates
 const SerialiserDeserialiser plainTextSerDes =
 {
-    .SerialiseObject   = NULL,
-    .SerialiseObjectInstance = NULL,
-    .SerialiseResource = PTSerialiseResource,
-    .DeserialiseObject = NULL,
-    .DeserialiseObjectInstance = NULL,
-    .DeserialiseResource = PTDeserialiseResource,
+    NULL,
```

Comment:
I'm wondering why this was ok when @datachi7d [built](https://github.com/datachi7d/AwaLWM2M/commit/6ffffb6d45f799345849bfbb81f53e1ab838a4d5#diff-73bfbb5a9a11ac3edaf5f4eebbdc4d99R294) this.  Must be a different version of tools?  What version tools are you using @francois-berder-imgtec ?